ENTITY/GetEntityCanBeDamaged.md

```diff
@@ -9,8 +9,10 @@ aliases: ["0xD95CC5D2AB15A09F"]
 BOOL GET_ENTITY_CAN_BE_DAMAGED(Entity entity);
 ```
 
+Getter for [`SET_ENTITY_CAN_BE_DAMAGED`](_0x1760FFA8AB074D66)
```

Comment:
Slight nitpick, this is supposed to have a preceding `#` before the underscore `_` i.e. `#_0x1760FFA8AB074D66`. Noticed this multiple times within this PR, missed it a couple times.  Not adding the hash will cause a 404 when clicked.

Reply:
I do believe you are correct.